tests/deployment/ipmi.py

```diff
@@ -55,19 +53,17 @@ def server_init():
 class TestIpmi():
     """ Test IPMI driver calls using functions included in the IPMI driver. """
 
-    def collect_nodes(self, db):
+    def collect_nodes(self):
         """Collects nodes in the free list."""
-        free_nodes = db.query(Node).filter_by(project_id=None).all()
+        free_nodes = Node.query.filter_by(project_id=None).all()
```

Comment:
Quoting henn (2016-04-05 01:03:36)

>    Where are we importing Node from?

Good question! I was just mechanically making the change, and assumed
since it was being referred to as `Node` already, the change was safe. I
bet this is broken. @SahilTikale, you wrote this, can you weigh in?